src/sidebar/components/moderation/ModerationStatusBadge.tsx

```diff
@@ -0,0 +1,40 @@
+import type { ModerationStatus } from '@hypothesis/annotation-ui';
+import { moderationStatusInfo } from '@hypothesis/annotation-ui';
+import classnames from 'classnames';
+
+export type ModerationStatusBadgeProps = {
+  status: ModerationStatus;
+  classes?: string | string[];
+};
+
+/**
+ * Badge representation of a moderation status, with the corresponding color
+ * palette, icon and text
+ */
+export default function ModerationStatusBadge({
+  status,
+  classes,
+}: ModerationStatusBadgeProps) {
+  // We only support these two moderation statuses for now
+  if (!['PENDING', 'DENIED'].includes(status)) {
+    return null;
+  }
```

Comment:
We are not rendering badges for approved annotations, and it's still to be confirmed if spam annotations do not show the badge either.

Because of that it decided to implicitly render "nothing" for those statuses here, rather than having to check every time this component is render, because then we don't need to implement styles/classes for those statuses.